### PR TITLE
18.1 upgrade

### PR DESCRIPTION
**Pull request by jeffpsherman**

### CODE REVIEW

File: GuaranteedRate.Util.IndexFields/GuaranteedRate.Util.IndexFields.csproj [hunk 1]
```diff
@@ -34,150 +34,162 @@
...
+      <HintPath>..\packages\EncompassSDK.Complete.18.1.0.1\lib\net45\Server.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="SharpZipLib, Version=0.4.0.0, Culture=neutral, PublicKeyToken=fbebc9694da332b7">
+      <HintPath>..\packages\EncompassSDK.Complete.18.1.0.1\lib\net45\SharpZipLib.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Web.Razor, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (GeorgeGomez) [inline GuaranteedRate.Util.IndexFields/GuaranteedRate.Util.IndexFields.csproj]: Razor version 2.0 ?

File: GuaranteedRate.Examples.LoanDataUtils/app.config [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-11.0.0.0" newVersion="11.0.0.0" />
```
Comment (HarryCordewener) [inline GuaranteedRate.Examples.LoanDataUtils/app.config]: Danger, Will Robinson!
Are we sure we are wanting to go above the version 10? We appear to be versioning to 10 with the SDK.

File: GuaranteedRate.Sextant/EncompassUtils/SessionUtils.cs [hunk 3]
```diff
@@ -30,8 +30,17 @@ public static Loan OpenLoan(Session session, string guid)
...
-            return session.Loans.Open(guid);
+            try
+            {
+                var loan = session.Loans.Open(guid);
+                return loan;
+                //return session.Loans.Open(guid);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```
Comment (HarryCordewener) [inline GuaranteedRate.Sextant/EncompassUtils/SessionUtils.cs]: Is Console.Writeline fine here? Can Sextant use its own logging here?

File: GuaranteedRate.Sextant/EncompassUtils/SessionUtils.cs [hunk 4]
```diff
@@ -30,8 +30,17 @@ public static Loan OpenLoan(Session session, string guid)
             {
                 guid = guid + "}";
             }
-
-            return session.Loans.Open(guid);
+            try
+            {
+                var loan = session.Loans.Open(guid);
+                return loan;
+                //return session.Loans.Open(guid);
```
Comment (HarryCordewener) [inline GuaranteedRate.Sextant/EncompassUtils/SessionUtils.cs]: Remove wayward comment.

**Pull request merged**